### PR TITLE
boards/nucleo-l452re: doc update

### PR DESCRIPTION
**Pull request by krzysztof-cabaj**
### Contribution description

This PR adds to the `nucleo-l452re` board documentation page MCU table with links to datasheets.

The issue was detected during working on PR #20071. 

### Testing procedure

Generate documentation and see if everything is OK.

```
make doc
xdg-open doc/doxygen/html/group__boards__nucleo-l452re.html 
```
### Issues/PRs references

Detected during works on PR #20071

### CODE REVIEW

File: boards/nucleo-l452re/doc.txt [hunk 1]
```diff
@@ -8,6 +8,29 @@
 The Nucleo-L452RE is a board from ST's Nucleo family supporting ARM Cortex-M4
 STM32L452RE microcontroller with 160KiB or RAM and 512KiB of Flash.
```
Comment (mguetschow) [inline boards/nucleo-l452re/doc.txt:9]:
```suggestion
STM32L452RE microcontroller with 160KiB of RAM and 512KiB of Flash.
```

maybe, while at it, you can fix this typo

**Review by mguetschow — CHANGES_REQUESTED** [review]
Nice addition, thanks! Just a minor comment below.

File: boards/nucleo-l452re/doc.txt (hunk 1 above)
Comment (krzysztof-cabaj) [reply boards/nucleo-l452re/doc.txt]: Fixed!

Thanks for spotting this typo.

**Review by mguetschow — APPROVED** [review]
Thanks LGTM then!

**Pull request merged**